pymc3/distributions/continuous.py

```diff
@@ -2548,7 +2549,7 @@ def logcdf(value, alpha, beta):
         )
 
 
-class ChiSquared(Gamma):
```

Comment:
We don't have to change the name of the PyMC3 distribution (unless that's want we want of course).

Reply:
I don't have a strong preference but if we change the name we should include the old class distribution just to raise a deprecation warning saying that the name has changed.